### PR TITLE
nbformat 5.1.2, add tests, maintainer

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

- alternative to #22 which is _probably broken_
- alternative to #25 (but also adds):
  - runs test suite with a `70` coverage threshold :tada: 
  - adds self as maintainer :wave:
  - adds docs site, other meta

Will also put out the admin-request to mark `5.1.0` broken...

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/nbformat ready for review!

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
Bumped the build number: it's probably still worth adding tests, even though we got #25 in to deal with #24.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {% set version = "5.1.2" %}
+# TODO: investigate upstream (whether it is expected these should exist)
+{% set k_skips = ["upgrade_v4_to_4_dot_5", "sample_notebook"] %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Do these need to be jinja vars? I'm not a big fan of those and they'll go away in the recipe format.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -49,3 +60,4 @@ extra:
     - takluyver
     - ocefpaf
     - mpacer
+    - bollwyvl
```
Comment (ocefpaf) [inline recipe/meta.yaml:61]: :tada:

**Comment by bollwyvl** [comment]
Removed Jinja vars. I despair of them going away entirely unless we have
skylark, dhall, json-e or _something_ to replace them that wasn't invented
just for conda.

**Review by ocefpaf — APPROVED** [review]

**Pull request merged**